tracks/create-actions.es6

```diff
@@ -122,20 +140,42 @@ export default function createActions(nodes, mountPoint='tracks') {
     }
   }
 
+  const UNMUTE = `${mountPoint}/UNMUTE`;
+  function unMute(id) {
+    return function unMuteThunk(dispatch, getState) {
+      const track = dotKey(mountPoint, getState())[id];
+
+      if (track && track.isReady && track.isPlaying) {
+        nodes.unmute(id);
```

Comment:
Yeh thanks - my bad! Howler had documentation saying they used unmute so wanted to keep consistent with, but in v 2 is mute(true/false)